Alloy/Renderers/PointsRenderer.swift

```diff
@@ -0,0 +1,166 @@
+//
+//  PointsRenderer.swift
+//  Alloy
+//
+//  Created by Eugene Bokhan on 26/04/2019.
+//
+
+import Metal
+import simd
+
+@available(iOS 11.3, tvOS 11.3, macOS 10.13, *)
+final public class PointsRenderer {
+
+    public enum Errors: Error {
+        case functionCreationFailed
+        case libraryCreationFailed
+        case wrongRenderTargetTextureUsage
+        case missingRenderTarget
+    }
+
+    // MARK: - Properties
+
+    /// Lines described in a normalized coodrinate system to draw.
+    public var points: [SimplePoint]? {
+        set {
+            if let newValue = newValue {
+                var convertedToMetalPoints = newValue
+                    .map { SimplePoint(position: packed_float2(x: -1 + ($0.position.x * 2),
+                                                               y: -1 + ((1 - $0.position.y) * 2)),
+                                       size: $0.size) }
```

Comment:
why store size with each point if we only support rendering uniformly sized points?